### PR TITLE
Merging chisel3_port into master

### PR DESCRIPTION
**Pull request by tianrui-wei**
Sequential Equivalence Checking using jaspergold shows that the current chisel3 port `f8d19c289942d8118048a816cb323ae5270380a8` is equivalent to `7c1f50691950e43e530b14263cf2706983a5a9f8`

### CODE REVIEW

**Review by sequencer — APPROVED** [review]
Thanks @tianrui-wei, since #3095(b70600b1d) and #3096(d613bf15c) is merged today morning, I reset `chisel3_port` to f8d19c289942d8118048a816cb323ae5270380a8, and approved this PR, after CI merging it, I will rebase #3095(b70600b1d) and #3096(d613bf15c) to the new master.

**Comment by tianrui-wei** [comment]
Sounds great!

**Pull request merged**
